<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgConfig</class>
 <widget class="QDialog" name="CDlgConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>191</width>
    <height>294</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>191</width>
    <height>294</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>191</width>
    <height>294</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>参数配置</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>20</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>主界面</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbxUIType"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="btnOk">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnClose">
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>49</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>识别源</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbxVideoType"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>77</y>
     <width>150</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>车牌路数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spPlateWay">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>4</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>104</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>网络相机IP</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtIpcIP"/>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="chxMainStream">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>132</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>网络相机主码流</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chxVideoCapture">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>150</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>视频抓图识别</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chxPlateThread">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>170</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>车牌识别多线程方式</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
